.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,7 @@
 ## If you are releasing a new version of notify, what components are you updating
 - [ ] API
 - [ ] Admin
+- [ ] Documentation
```

Comment:
I've observed this was missing from our components list.

Reply:
Thanks for adding it, I've been wanting to do that for a while and I've been lazy.